src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -279,13 +279,25 @@ def fuzz(
       engine.FuzzResult
     """
 
-    def _filter_log(content):
-      """Filter unneeded content from log."""
-      result = ''
-      strip_regex = re.compile(r'^c\d+\s+\d+\s')
-      for line in content.splitlines():
-        result += strip_regex.sub('', line) + '\n'
-      return result
+    def _filter_log(content: str) -> str:
+      """Remove unnecessary prefix from each line of log.
+
+      e.g
+      [  565.723853] c4   8262 BUG: KASAN: ...
+      vs.
+      BUG: KASAN: ...
+
+      Args:
+        content (str): log content
+      Returns:
+        filtered log with new lines (str)
+      """
+
+      strip_regex = re.compile(r'^(\[.*\]\s+)?c\d+\s+\d+\s')
```

Comment:
nit: it might be better to use `.*?` instead (non greedy) so that the matching stops at the first ']'.

Reply:
I see. I thought regex would match only once? so `\]` would always match the first `]`? Is this accounting for a case with multiple closing brackets?

Reply:
Yes, .* is greedy by default and will as much as it can.

```
>>> re.match(r'(\[.*\])', '[foo] [bar]').group(1)
'[foo] [bar]'

>>> re.match(r'(\[.*?\])', '[foo] [bar]').group(1)
'[foo]'
```

Reply:
thanks for the feedback. this is fixed.